src/PhpPresentation/Writer/PowerPoint2007/PptSlides.php

```diff
@@ -757,6 +828,7 @@ protected function writeSlideAnimations(XMLWriter $objWriter, Slide $oSlide)
      */
     protected function writeShapeDrawing(XMLWriter $objWriter, ShapeDrawing\AbstractDrawingAdapter $shape, $shapeId)
     {
+        exit( 'eyah');
```

Comment:
Please remove it

Reply:
@Progi1984 The reason I had this debug message here, is because I noticed that this function, `writeShapeDrawing` is never called anywhere. Would it be a good idea to remove it entirely?